curvefs/src/client/inode_cache_manager.h

```diff
@@ -272,7 +277,11 @@ class InodeCacheManagerImpl : public InodeCacheManager,
 
     void RemoveOpenedInode(uint64_t inodeId) override;
 
-    bool NeedUseCahce(uint64_t inodeId, bool IsDirty);
+    bool NeedUseCahce(uint64_t inodeId,
+        const std::shared_ptr<InodeWrapper> &inodeWrapper,
+        bool onlyAttr);
```

Comment:
What does `onlyAttr` mean here.

Reply:
If the inode dirty you need return from cache, the dirty have two layer: attr dirty and data dirty.
This flag is used to mark the request only need attr from inode, so if the attr is clean and timeout, should get from metaserver even if data is dirty, the refreshed inodeAttr will not set to cache nowadays.